final_project/Analyses/HW04/HW04.md

```diff
@@ -0,0 +1,85 @@
+HW04
+================
+Han Lee
+3/5/2021
+
+``` r
+#Reading in data file and making sure all columns are correct and controlled for.
+dat <- read_csv(here::here("raw_data","RepeatedExposureReverseRdata-3_13_2020.csv"),
+                col_types = cols(
+                  `F/L/Sex` = col_character(),
+                  ID = col_double(),
+                  TrialType = col_character(),
+                  `Trial#` = col_double(),
+                  `RunPosition#` = col_double(),
+                  `Run#` = col_double(),
+                  ConditionType = col_character(),
+                  Condition = col_character(),
+                  RT = col_double(),
+                  Error = col_double(),
+                  TotalError = col_double(),
+                  Target = col_character(),
+                  Globality = col_character(),
+                  RunType = col_character(),
+                  StimuliType = col_character()
+                  ),
+                  na = c("","NA")
+                ) %>% 
+    filter(TrialType == "Experiment", ConditionType == "Critical", Error == "0", (RT >= 50 & RT <= 1999)) #Filtering out to relevant data (i.e., cleaning raw data)
+
+#Creating clean csv file from filtering
+write_csv(dat,
+          here::here("Analyses", "HW04", "Data", "HW04_cleaned_data.csv")
+          )
+```
+
+``` r
+#Histogram of RT distribution
```

Comment:
Instead of commenting inside the code chunk, try to comment the details outside of it!